src/components/Incorporation/IncorporationDateTime.vue

```diff
@@ -152,21 +154,21 @@ export default class IncorporationDateTime extends Mixins(DateMixin) {
   }
 
   /** The array of validations rules for effective date hours field. */
-  private get hourRules (): Array<Function> {
+  get hourRules (): Array<Function> {
     return [
       v => this.dateText !== '' && (/^([1-9]|1[012])$/.test(v) || '')
```

Comment:
This _cannot_ be a readonly array because it uses a variable in its code. So it's a getter instead.